salt/journal-cms/init.sls

```diff
@@ -148,7 +148,18 @@ site-settings:
 journal-cms-{{ key }}:
     mysql_database.present:
         - name: {{ db.name }}
+        {% if salt['elife.cfg']('cfn.outputs.RDSHost') %}
+        # remote mysql
+        - connection_user: {{ salt['elife.cfg']('project.rds_username') }} # rds 'owner' uname
+        - connection_pass: {{ pillar.elife.db_root.password }} # rds 'owner' pass
```

Comment:
Lax uses a generated RDS password https://github.com/elifesciences/lax-formula/blob/master/salt/lax/init.sls#L82 at least to create a non-root user; once the non-root user is there it can be used for all further operations. Why the inconsistency?